mmv1/provider/terraform_tgc_next.go

```diff
@@ -310,6 +317,54 @@ func (tgc TerraformGoogleConversionNext) replaceImportPath(outputFolder, target
 	}
 }
 
+func (tgc TerraformGoogleConversionNext) addExamplesFromHandwrittenTests(object *api.Resource) error {
+	if object.ProductMetadata == nil {
+		return nil
+	}
+	product := object.ProductMetadata
+	productName := google.Underscore(product.Name)
+	resourceFullName := fmt.Sprintf("%s_%s", productName, google.Underscore(object.Name))
+	handwrittenTestFilePath := fmt.Sprintf("third_party/terraform/services/%s/resource_%s_test.go", productName, resourceFullName)
+	data, err := os.ReadFile(handwrittenTestFilePath)
+	for err != nil {
+		if errors.Is(err, os.ErrNotExist) {
+			if strings.HasSuffix(handwrittenTestFilePath, ".tmpl") {
+				log.Printf("no handwritten test file found for %s", resourceFullName)
+				return nil
+			}
+			handwrittenTestFilePath += ".tmpl"
+			data, err = os.ReadFile(handwrittenTestFilePath)
+		} else {
+			return fmt.Errorf("error reading handwritten test file %s: %v", handwrittenTestFilePath, err)
+		}
+	}
+
+	// Skip adding handwritten tests that are already defined in yaml (because they have custom overrides etc.)
```

Comment:
I wonder how will the custom overrides look like in yaml. We may see an example in later PRs?